.docker/parameters.yml

```diff
@@ -39,6 +39,7 @@ parameters:
               - '26231'
     cache_control: private, no-cache, no-store, must-revalidate
     feature_xpub: true
+    feature_rds: true
```

Comment:
could be more specific as some RDS implementation is already there and deactivating this flag won't hide it?

Reply:
This replaces that implementation so will hide it.

Reply:
https://github.com/elifesciences/journal/pull/1220#discussion_r374736230 reverses that.